### ISSUE TITLE
Pasting content with `--expect`-ed keys

### ISSUE THREAD

**Issue by jbriales** [junegunn/fzf#1803]
- [x] I have read through the manual page (`man fzf`)
- [x] I have the latest version of fzf
- [x] I have searched through the existing issues

## Info

- OS
    - [x] Linux
    - [ ] Mac OS X
    - [ ] Windows
    - [ ] Etc.
- Shell
    - [x] bash
    - [ ] zsh
    - [ ] fish
    
## Problem / Steps to reproduce
e.g. `fzf --expect=space`
copy 'foo bar' and paste within fzf

When pasting from clipboard content with multiple spaces only content up to first space is pasted due to `--expect=space`

In general I guess this is an issue with any `--expect`-ed characters when pasted.
I think it might make more sense that pasted content is treated as general text rather than caught as control commands by expect.
Is there any workaround for this?
Does it make sense this is the default behavior?
Otherwise, how hard would it be to change this behavior? :)

**Comment by 0ion9** [comment]
Pastes are equivalent to the user typing the series of characters which are on the clipboard.
Some terminal emulators produce special characters bracketing the pasted content, which in theory can be used to treat pasted content differently. These sequences seem to be `\e[200\~` and `\e[201~` respectively.

I haven't found a reliable way to test which terminal emulators produce these, unfortunately.

**Comment by junegunn** [comment]
> Pastes are equivalent to the user typing the series of characters which are on the clipboard.

Yes, this is on point, and I don't want to implement a partial solution that may or may not work depending on the user's environment. I suggest that you choose a different bind key that can't appear in normal character sequences.

**Comment by jbriales** [comment]
Found my workaround for this! :D
This happened in the context of a fzf-based app I have, and having space as a bind key was necessary to have the desired refresh-rate in sourcing/updating the fzf input.

Anyway, I found a pretty decent workaround for me, sharing in case it may be of interest to others who come here:
- Add `--expect=ctrl-v` to finish fzf upon hitting Ctrl-V
- Check the hitted key was `ctrl-v`
- Call `fzf` again populating the query from the clipboard:
  `fzf ... --query="$(xclip -o)"`
- Success! :D

Now if I want to paste a line containing spaces in fzf I just need to do Ctrl-V :)
